test/lib/dashboard-agent.spec.js

```diff
@@ -64,18 +66,38 @@ describe("dashboard-agent", function () {
 
       var checkMetrics = function (metrics) {
         expect(metrics).to.be.an("object");
-        expect(metrics.eventLoop).to.deep.equal({ delay: 0, high: 0 });
-        expect(metrics.mem.systemTotal).to.be.above(0);
-        expect(metrics.mem.rss).to.be.above(0);
-        expect(metrics.mem.heapTotal).to.be.above(0);
-        expect(metrics.mem.heapUsed).to.be.above(0);
-        expect(metrics.cpu.utilization).to.be.above(0);
+        expect(metrics.eventLoop.delay).to.be.a("number");
```

Comment:
GLOBAL SUGGESTION: I know this is a refactor, but for things like:

```js
expect(metrics.eventLoop.delay).to.be.a("number");
```

if for example, `metrics.eventLoop` is `undefined`, you're going to get a not-so-useful assertion failure. A better way to do this is to assert on the property and let chai tell you "that deep property doesn't exist". Aka

```js
expect(metrics).to.have.deep.property("eventLoop.delay").that.is.a("number");
```

(Side note -- unique to `.property` assertions, the context changes from there on to the _property_ asserted against (e.g. `metrics.eventLoop.delay`) and not the original target object (e.g. `metrics`))